codechain/keystore/HDStorageJson.py

```diff
@@ -0,0 +1,88 @@
+import binascii
+import os
+import uuid
+
+from Crypto.Cipher import AES
+from Crypto.Util import Counter
+
+from ..crypto import blake256
+from ..crypto import get_public_from_private
+from .Errors import ErrorCode
+from .Errors import KeystoreError
+from .keys import key_from_public_key
+from .KeyType import KeyType
+from .Pbkdf2 import pbkdf2
+
+
+def encode(seed: str, passphrase: str, meta: str):
+    seed_hash = blake256(seed)
+    salt = os.urandom(32)
+    iv = os.urandom(16)
+
+    kdf = "pbkdf2"
+    kdf_params = {
+        "dklen": 32,
+        "salt": binascii.hexlify(salt).decode("ascii"),
+        "c": 262144,
+        "prf": "hmac-sha256",
+    }
+    derived_key = pbkdf2(
+        passphrase.encode("utf-8"),
+        salt,
+        kdf_params.get("c"),
+        kdf_params.get("dklen"),
+        "sha256",
+    )
+    ctr = Counter.new(128, initial_value=int_of_string(iv))
```

Comment:
`iv` goes through following code path. It is converted into `str` and then to `int` while you can use `int.from_bytes`.
```
os.urandom(16)
|> binascii.hexlify(_)
|> int(_, 16)
```

Reply:
I fixed it.